### PR TITLE
Go command read in index instead of module name

### LINKED ISSUE #153: [PE-D] Instead of go <name>, Can use go <index>.
I think 'go [index]' can a much faster option for the user and fast typist since it requires less effort and easier typing.

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: johanesrafael/ped#7

### PR DESCRIPTION
**Pull request by gua-guargia**
fixes #153

### CODE REVIEW

File: src/main/java/commands/GoChapterCommand.java [hunk 1]
```diff
@@ -31,32 +38,29 @@ public void execute(Ui ui, Access access, Storage storage) {
...
-                } catch (FileNotFoundException e) {
-                    result = "The chapter file cannot be found.";
-                }
-                break;
+        try {
+            ChapterList chapters = access.getModule().getChapters();
+            ArrayList<Chapter> allChapters = chapters.getAllChapters();
+            Chapter chapter = allChapters.get(chapterIndex);
+            logger.info("Going into chapter: " + chapter.toString());
+            access.setChapterLevel(chapter.getChapterName());
```
Comment (Jane-Ng) [inline src/main/java/commands/GoChapterCommand.java]: this line can be shift to after line 54: access.setChapter(chapter)
so you load the cards first. and if the chapter file cannot be found then access wont set anything yet
so access.setChapterLevel should be after loading the cards

File: src/main/java/commands/GoModuleCommand.java [hunk 2]
```diff
@@ -32,32 +40,29 @@ public void execute(Ui ui, Access access, Storage storage) throws IOException {
...
-                } catch (FileNotFoundException e) {
-                    result = "The module folder cannot be found.";
-                }
-                break;
+        try {
+            ModuleList modules = access.getAdmin().getModules();
+            ArrayList<Module> allModules = modules.getAllModules();
+            Module module = allModules.get(moduleIndex);
+            logger.info("Going into module: " + module.toString());
+            access.setModuleLevel(module.getModuleName());
```
Comment (Jane-Ng) [inline src/main/java/commands/GoModuleCommand.java]: this line can be shift to after line 56: access.setModule(module)
so you load the chapters first. and if the module cannot be found then access wont set anything yet
so access.setModuleLevel should be after loading the chapters

File: src/main/java/parser/GoCommandParser.java [hunk 3]
```diff
@@ -1,42 +1,59 @@
...
+import static common.Messages.MESSAGE_MISSING_INDEX;
+import static common.Messages.MESSAGE_NON_INTEGER;
 import static common.Messages.MODULE;
 
 //@@author gua-guargia
 public class GoCommandParser {
-    private static final String MESSAGE_INCORRECT_ACCESS = "Go command can only be called at admin and module level.\n";
-
-    public Command parse(String commandArgs, String accessLevel)
+    public GoCommand parse(String commandArgs, String accessLevel)
```
Comment (Jane-Ng) [inline src/main/java/parser/GoCommandParser.java]: i think you can leave GoCommand as Command. but if its working then its fine

Comment (gua-guargia) [reply src/main/java/parser/GoCommandParser.java]: Thank you! Have changed it to Command

**Pull request merged**